ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -697,6 +697,10 @@ impl TraceBuilder {
         if inst.is_debug_call(self.aot_mod) {
             return Ok(());
         }
+        // Ignore software tracer calls. Software tracer inserts
+        // `yk_trace_basicblock` insturction calls into the begining of
+        // every basic block. These calls can be ignored as they only
+        // used to collect runtime information for the tracer itself.
```

Comment:
There's a few typos here: s/insturction/instruction, s/begining/beginning, s/they only/they are only. There's no need to push a new commit. Just fix them during squashing.

Reply:
oops.
fixed :point_right: 541e8ade5bc8742c9e06547afef10d99bdde8e41